pkg/fs/fs-bucket-listobjects.go

```diff
@@ -60,8 +64,9 @@ func (fs Filesystem) ListObjects(bucket, prefix, marker, delimiter string, maxKe
 		}
 
 		if !strings.HasPrefix(marker, prefix) {
-			return result, probe.NewError(fmt.Errorf("marker '%s' and prefix '%s' do not match", marker, prefix))
+			return result, probe.NewError(fmt.Errorf("marker '%s' and prefix '%s' fails to satisy the check strings.HasPrefix(%s,%s)", marker, prefix, marker, prefix))
```

Comment:
This message is too low level "fails to satisfy strings.HasPrefix" is wrong.  Messages should be user facing and not about code but context.